### ISSUE TITLE
Allow people to sign up to the application

### ISSUE THREAD

**Issue by ajones5876** [yldio/joyent-portal#200]
## Problem Statement
A user needs to sign up to the application

## Persona
All

## Latest Links (kept up to  date)
Initial research: https://docs.google.com/document/d/1YAzTKA50drT4O3zkscBDF1NzIiIo2ywLYUL_OAvWMco/edit
Workshop Sketches: N/A
Visual Design: https://drive.google.com/open?id=0B1oWObk56wa5ZjFuMVFCUnpSOFE
Prototype: https://marvelapp.com/1701gf9/screen/17500333 (out dated)
Test Results: https://docs.google.com/document/d/165agJVUiNxIkk90wDEZhYNBR8WoQpE7v1WdBneklqAU/edit (out dated)

**Comment by ajones5876** [comment]
**Prototype**
https://marvelapp.com/1d5902h/screen/24487210

**Comment by ajones5876** [comment]
**New Two Factor Authentication**
![image](https://cloud.githubusercontent.com/assets/7900773/22788692/8c52ae7c-eed8-11e6-9b33-a877cd7266b9.png)

![image](https://cloud.githubusercontent.com/assets/7900773/22788703/93a4b4b8-eed8-11e6-8712-4b49a575df84.png)

**Comment by ajones5876** [comment]
**New Visual Designs, Prototype and Testing Results**
- Prototype: https://marvelapp.com/1jgi1ca
- Test Results: https://drive.google.com/open?id=1bSS67EAeNObldIJil3ZoUQEJFDGDsOKxI_mrLe57HSw

The new designs incorporate:
- a change in the billing details design
- addition of sign in (along with sign up)
- illustrations for branding
![image](https://cloud.githubusercontent.com/assets/7900773/24206521/5e35ee24-0f16-11e7-9267-57072079be57.png)
![image](https://cloud.githubusercontent.com/assets/7900773/24206535/6a49ec10-0f16-11e7-9da1-1cd0d9af9e04.png)
![image](https://cloud.githubusercontent.com/assets/7900773/24206550/73fedcac-0f16-11e7-8a7e-d1184f0fc1d1.png)

**Testing Feedback**
Comments from testing sessions are recorded here: https://drive.google.com/open?id=1bSS67EAeNObldIJil3ZoUQEJFDGDsOKxI_mrLe57HSw
In summary:
- the reception to the illustrations has been non-existant or luke warm
- some questions were raised about the implementation of password managers
- some confusion around having to specify an already inputted card for a project
- positive reactions to two-factor authentication
- in general testing subjects responded well to this sign up flow
